### PR TITLE
[2.1] Revert all nightly tests change related to bundle blocks feature

### PR DESCRIPTION
**Pull request by c21**
## Why are these changes needed?

As title, all nightly test changes are

https://github.com/ray-project/ray/commit/928c528075a44a42c5b233bc9fbbaef72b54adc1
https://github.com/ray-project/ray/commit/4605b58b932e03cf0eb89592be82be7c9a72d1bc
https://github.com/ray-project/ray/commit/f017a905275ae0418d914a99c6808e551f969a20

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/nightly_tests/dataset/inference.py [hunk 1]
```diff
@@ -93,10 +93,7 @@ def infer(batch):
 ds = ds.map(preprocess)
 end_preprocess_time = time.time()
 print("Inferring...")
-# NOTE: set a small batch size to avoid OOM on GRAM when doing inference.
-ds = ds.map_batches(
-    infer, num_gpus=0.25, batch_size=128, batch_format="pandas", compute="actors"
-)
+ds = ds.map_batches(infer, num_gpus=0.25, batch_format="pandas", compute="actors")
```
Comment (c21) [inline release/nightly_tests/dataset/inference.py:96]: This is change from my PR.

File: release/air_tests/air_benchmarks/workloads/gpu_batch_prediction.py [hunk 2]
```diff
@@ -43,9 +43,7 @@ def main(data_size_gb: int):
     ckpt = TorchCheckpoint.from_model(model=model, preprocessor=preprocessor)
 
     predictor = BatchPredictor.from_checkpoint(ckpt, TorchPredictor)
-    predictor.predict(
-        dataset, num_gpus_per_worker=1, feature_columns=["image"], batch_size=512
-    )
+    predictor.predict(dataset, num_gpus_per_worker=1, feature_columns=["image"])
```
Comment (c21) [inline release/air_tests/air_benchmarks/workloads/gpu_batch_prediction.py:46]: Please double check @amogkam, thanks.

**Comment by c21** [comment]
btw are there any other nightly tests change for `batch_size`, that you guys aware of? Went through the commits history, and that are all I found. Thanks.

**Comment by jiaodong** [comment]
> btw @amogkam and @jiaodong - are there any other nightly tests change for `batch_size`, that you guys aware of? Went through the commits history, and that are all I found. Thanks.

there's also one from xgboost https://github.com/ray-project/ray/pull/29394

**Comment by c21** [comment]
@jiaodong - yeah https://github.com/ray-project/ray/pull/29394 is already covered here. Do you mean to provide another different PR link?

**Review by amogkam — APPROVED** [review]
lgtm. can we rerun the tests before merging?

**Comment by c21** [comment]
Thanks @amogkam and @jiaodong. Will rerun test before merging.

**Comment by c21** [comment]
Looks like there's no easy way to trigger the nightly test in release branch for this PR. Talked to @amogkam offline, will merge this PR first and run release test in release branch.

**Pull request merged**